Node.js APM docs updates (#9333)
DataDog/documentation: content/en/tracing/setup_overview/setup/nodejs.md

## Installation and getting started

> To add the Datadog tracing library to your Node.js applications, follow the 4 steps below.

Reviewer: ```suggestion
To add the Datadog tracing library to your Node.js applications, follow these steps:
```

1. Install the Datadog Tracing library using npm:

```sh
npm install --save dd-trace
```

2. Import and initialize the tracer either in code or via command line arguments. The Node.js tracing library needs to be imported and initialized **before** any other module.

>    Once you have completed setup, if you are not receiving complete traces, including missing url routes for web requests, disconnected or missing spans, **confirm step 2 has been correctly done**.  The tracing library being initialized first is necessary for the tracer to properly patch all of the required libraries for automatic instrumentation.

Reviewer: ```suggestion
   Once you have completed setup, if you are not receiving complete traces, including missing URL routes for web requests, or disconnected or missing spans, **confirm step 2 has been correctly done**.  The tracing library being initialized first is necessary for the tracer to properly patch all of the required libraries for automatic instrumentation.
```

>    When using a transpiler such as `TypeScript`, `webpack`, `babel` or others, import and initialize the tracer library in an external file and then import that file as a whole when building your application.

Reviewer: ```suggestion
   When using a transpiler such as TypeScript, Webpack, Babel, or others, import and initialize the tracer library in an external file and then import that file as a whole when building your application.
```

3. [Configure the Datadog Agent for APM](#configure-the-datadog-agent-for-apm)

> 4. Add any desired [configuration](#configuration) to the tracer, such as `service`, `env` and `version` for [Unified Service Tagging][2].

Reviewer: ```suggestion
4. Add any desired [configuration](#configuration) to the tracer, such as `service`, `env`, and `version` for [Unified Service Tagging][2].
```

const tracer = require('dd-trace').init();
```

> #### TypeScript and Bundlers

Reviewer: ```suggestion
#### TypeScript and bundlers
```
